### ISSUE TITLE
A11y: Add containerRole parameter

### ISSUE THREAD

**Issue by michaelbourne** [nolimits4web/swiper#7705]
### Clear and concise description of the problem

The Accessibility module currently allows for a `aria-roledescription` attribute on the slider container via the   [containerRoleDescriptionMessage](https://swiperjs.com/swiper-api#param-containerRoleDescriptionMessage) option, but there is no support for setting a `role` to the same container.

As per https://www.w3.org/TR/wai-aria-1.1/#aria-roledescription, this is a slight violation of best practices, and my a11y auditors have flagged it as an issue to be fixed.

### Suggested solution

Add a `containerRole` parameter to the Accessibility module so we can set the proper role (`region`) when setting the `containerRoleDescriptionMessage` option.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
What is the issue? What does it do? What is it for? What does it fix? Why you can't just add `aria-role` with "hands" to HTML element?

Your issue is closed because it is not filled correctly. Before you open an issue please review the [contributing guideline](https://github.com/framework7io/framework7/blob/master/CONTRIBUTING.md).

Please fill the issue template correctly and:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with isssue

It will save a lot of time for other contributors to check your issue and fix it.
